modules/querying/pages/index.adoc

```diff
@@ -1,12 +1,37 @@
 
 This document focuses on the formal specification for the GSQL Query Language. It includes example queries that demonstrate the language, each of which works on one of the following six graphs:**`workNet`, `socialNet`, `friendNet`, `computerNet`, `minimalNet`,** and *`investmentNet`* . Their schemas are shown below. Appendix D lists the full command and data files to create and load these graphs with small sets of data (~10 to 20 vertices). The data sets are small so that you can understand the result of each query example. The tarball file *`gsql_ref_examples_2.0.tar.gz`* (linked below) contains all of the graph schemas, data files, and queries.
 
 link:{attachmentsdir}/gsql_ref_examples_2.0.tar.gz[Download Graph files and loading script].
 
-== Schemas for Example Graphs
+== GSQL syntax versions
+There are now two versions of the GSQL Query Language. Both are available in the product.
+The original version is called GSQL Syntax V1.
+GSQL Syntax V2 was introduced to provide more flexible and powerful xref:tutorials:pattern-matching/README.adoc[Pattern matching].
+A query indicates which grammar to use in the SYNTAX clause of the `CREATE QUERY` header.
+V2 is the default.
+
+As new features are being added to GSQL, some are available only in V2.
+
+The key differences between the two grammars:
+
+. SQL-like Tabular `SELECT` is only in Syntax V2
+. Multi-hop Pattern Matching is only in Syntax V2
+** In V2, each `SELECT` statement can traverse a multi-hop path.
+*** The traversal direction is under the control of the query writer, with arrowheads on each edge set to show the direction.
+*** There is no arrowhead outside the parentheses:
++
+`Start:s -( (ForwardEdge> | <BackwardEdge):e )- Target:t`
+
+*** Users can write paths that explicitly show multiple hops, and they can use a Kleene star (`*`) to indicate repetition. `Start:s -(Edge1>:e1)- Middle:m -(<Edge2:e2)- Target:t` or `Start:s -(Edge*1..3)- Target:t`
+** In V1 (default), each `SELECT` statement can traverse one hop (from a set of vertices to their adjacent vertices).
+*** To write a multi-hop query, you write a sequence of `SELECT` statements.
+*** The traversal action is from left to right, and the notation uses "ASCII art" to depict a connection, either with a rightward facing arrowhead or no arrowhead: `+Start:s -( Edges:e )-> Target:t+` or  `Start:s -( Edges:e )- Target:t`
```

Comment:
The no-arrowhead case is one of the main sources of compatibility issue between v1 and v2. Therefore let's try to avoid mentioning the no-arrowhead case for v1.

Reply:
Hmm, I think this bullet point should stay still because it is describing V2 syntax differences, and the no arrowhead point is one of the major differences. The reality is that people who are used to writing queries in V1 are used to arrow heads, and need to told explicitly not to use them. 

To have users write less with arrowhead in V1, what I can do is remove the arrowhead in our V1 examples and explicitly recommend against using the arrowhead.